src/main/java/com/exasol/projectkeeper/AbstractProjectKeeperMojo.java

```diff
@@ -29,9 +29,12 @@
     @Parameter(property = "modules")
     private List<String> modules;
 
-    @Parameter(property = "excludeFiles")
+    @Parameter(property = "excludedFiles")
```

Comment:
How did it work before if in the documentation it's said 'excludedFiles'? Is it a bug?

Reply:
maven takes the name of the variable. I don't know what the property in the annotation is for...

Reply:
The property parameter can be used to allow configuration of the mojo parameter from the command line by referencing a system property that the user sets via the -D option.

From here: https://maven.apache.org/guides/plugin/guide-java-plugin-development.html

Maybe you can remove the `property` if you don't need them?

Reply:
I just tired and it makes the tests fail. So I'll just leave it as it is since maven seems to require it somehow.